README.md

```diff
@@ -11,12 +11,11 @@
 
 ## Getting Started
 
-VechainThor is the layer 1 blockchain, highly compatible† with Ethereum, which powers the vechain ecosystem.
-VechainThor is a public blockchain that is designed for mass adoption of blockchain technology by enterprise users of
+VechainThor is the layer 1 blockchain, highly compatible with EVM<sup>*</sup>, which powers the vechain ecosystem.
+VechainThor is a public blockchain that is designed for the mass adoption of blockchain technology by enterprise users of
 all sizes and is intended to serve as a foundation for a sustainable and scalable enterprise blockchain ecosystem.
 
-- † - VechainThor is currently up-to-date with the Ethereum's `Paris` hard fork, supporting Solidity version `0.8.18`.
```

Comment:
This one also has another option, using the github's footnotes
<img width="894" alt="Screenshot 2024-04-23 at 16 28 53" src="https://github.com/vechain/thor/assets/8411962/3c939956-6cc3-4c36-ba68-9ff38919dc09">